src/sessions/session_manual.cpp

```diff
@@ -189,15 +189,30 @@ void session_manual::attach_protocols(channel::ptr channel)
 }
 
 void session_manual::handle_channel_stop(const code& ec,
-    const std::string& hostname, uint16_t port)
+    const std::string& hostname, uint16_t port, uint32_t remaining,
+    channel_handler handler)
 {
     LOG_DEBUG(LOG_NETWORK)
         << "Manual channel stopped: " << ec.message();
 
     // Special case for already connected, do not keep trying.
     // After a stop we don't use the caller's start handler, but keep connecting.
     if (ec != error::address_in_use)
```

Comment:
For style I would invert this and return, reducing  depth.

Reply:
Assume you mean the `if (ec !=`.  Fixed.